<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="4" hostname="M5-D-7659NK2" name="com.RM.Main_Scripts.Aise_hi" tests="5" failures="0" timestamp="02 Feb 2018 09:22:54 GMT" time="10802.732" errors="1">
  <testcase name="Browser_set" time="10802.732" classname="com.RM.Main_Scripts.Aise_hi">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat com.RM.Generic.GenericMethods2.logfail(GenericMethods2.java:130)at com.RM.Generic.GenericMethods2.logFail(GenericMethods2.java:118)at com.RM.Main_Scripts.Aise_hi.Browser_set(Aise_hi.java:36)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:163)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </error>
  </testcase> <!-- Browser_set -->
  <testcase name="Menu_Navigation" time="0.000" classname="com.RM.Main_Scripts.Aise_hi">
    <skipped/>
  </testcase> <!-- Menu_Navigation -->
  <testcase name="Xnarf_OB" time="0.000" classname="com.RM.Main_Scripts.Aise_hi">
    <skipped/>
  </testcase> <!-- Xnarf_OB -->
  <testcase name="Xnarf_OB" time="0.000" classname="com.RM.Main_Scripts.Aise_hi">
    <skipped/>
  </testcase> <!-- Xnarf_OB -->
  <testcase name="tear" time="0.000" classname="com.RM.Main_Scripts.Aise_hi">
    <skipped/>
  </testcase> <!-- tear -->
</testsuite> <!-- com.RM.Main_Scripts.Aise_hi -->
